website/docs/d/pi_virtual_serial_numbers.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM: ibm_pi_virtual_serial_numbers"
+description: |-
+  Provides data for a virtual_serial_number in an IBM Power Virtual Server cloud.
+---
+
+# ibm_virtual_serial_numbers
```

Comment:
Doesn't the linter complain about the lack of newlines here? It does for me on other files that haven't been updated yet.